apps/web/lib/features/team/invite/user-invite-card.tsx

```diff
@@ -225,9 +225,9 @@ export function InviteUserTeamCard({
 					<DraggerIcon className="fill-[#CCCCCC] dark:fill-[#4F5662]" />
 				</div>
 
-				<div className="absolute opacity-40 right-2">
+				{/* <div className="absolute opacity-40 right-2">
```

Comment:
@YehualashetGit why this got commented out? Not needed anymore?

Reply:
yes we don't need this code s actually,  basically the invite row won't have such menu , may be i will remove it

Reply:
Yes, makes sense @YehualashetGit

Reply:
@YehualashetGit On the other hand, idea was to make that Invite row to look similar to other rows... Iike it's still "empty" but when you invite someone that person will be shown there with task and other info... So I would prefer you to restore it back :)))

Also in the future when you removing something (even if it looks like not used), please state it in the PR explicitly - we should be very careful with remove things if we not sure about initial idea why it was added etc.

Reply:
okay understood , the main reason why I removed it was , there was no real data to display and they are just a place holder , that's why , but will reverted it back

Reply:
@evereq  is this a temporary or it only be visible on Table view cause ,it'll now  make me to add more logic on TeamMeberTable View component to the invite row with separate actions